Adding information into the Helm Charts and Apps page to describe the… (#1243)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

1. In the left navigation menu on the **Cluster Dashboard**, click **Apps > Repositories**.
1. Use the toggle next to the **State** field to select all repositories, or toggle specified chart repositories to sync changes.
1. Click **Refresh**.

> 1. The **⋮** attached to the end of respective chart repositories also include a **Refresh** option which can be clicked to refresh the respective repository.

Reviewer: ```suggestion
1. The **⋮** at the end of each chart repository row also includes a **Refresh** option, which can be clicked to refresh the respective repository.
```

This is a little awkward, but I was trying to get all the nouns to the same state (singular), so that when we hit the verb there isn't any confusion on whether it should be singular or plural

Author: I had a bit of back and forth with myself on how to word this final step and your phrasing makes grammatical sense Marty, just updated across versions. Thanks for reviewing!
